Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:latest AS base
+FROM alpine:3.11 AS base
```

Comment:
[bigfiles] This file, one of the 3 largest in the project, increased in size to 212 lines.  The total size of those files is now 494 lines (target: 453).  Is this file complex enough to refactor?